locales/us.js

```diff
@@ -140,6 +140,7 @@ module.exports = function(locale, platform, userAgent) {
   }
   locale.bindKeyCode(semicolonKeyCode,    ['semicolon', ';']);
   locale.bindKeyCode(dashKeyCode,         ['dash', '-']);
+  locale.bindKeyCode(equalKeyCode, ['equal', 'equalsign', '=']);
```

Comment:
Can you align the array with the existing lines?

Reply:
Of course, no problem, thank you for your reminder.